docs/jwst/model_blender/main.rst

```diff
@@ -0,0 +1,105 @@
+.. _blender_handbook:
+
+Role of Model Blender
+======================
+
+One problem with combining data from multiple exposures stems from not being able
+to keep track of what kind of data was used to create the final product.  The 
+final product only reports one value for each of the metadata attributes from the
+model schema used to describe the science data, where each of multiple inputs may
+have had different values for each attribute.  The model_blender package solves
+this problem by allowing the user to define rules that can be used to determine a
+final single value from all the input values for each attribute, using separate 
+rules for each attribute as appropriate.  This package also creates a FITS binary
+table that records the input attribute values for all the input models used to 
+create the final product, allowing the user to select what attributes to keep in 
+this table.
+
+This code works by 
+
+  - reading in all the input datamodels (either already in-memory or from FITS files)
+  - evaluating the rules for each attribute as defined in the model's schema
+  - determining from definitions in the input model's schema what attributes to keep in the table
+  - applying each attributes rule to the set of input values to determine the final output value
+  - updating the output model's metadata with the new values
+  - generating the output table with one row for each input model's values
+  
+  
+Using model_blender
+===================
+The model blender package requires
+
+  - all the input models (or FITS files) be available
+  - the output product has already been generated
```

Comment:
Does the output product have to already exist in the form of a FITS file, or simply a data model in memory? I believe it's the latter, right?

Reply:
Actually, the code has been written to support both options for both inputs and output products.  As you can see in the commit, the documentation states that working directly with the datamodels is preferred and points out the I/O efficiencies as the primary advantage.  However, anyone wanting to run this code on data already processed and written out to disk will still be able to blend their data as a stand-alone, file-based operation.